### ISSUE TITLE
Cannot add validators to DateTimeField 

### ISSUE THREAD

**Issue by joeljames** [localmed/pyserializer#25]
```
class SomeDeserializer(Serializer):
        time = fields.DateTimeField(
            validators=[validators.RequiredValidator()],
           format='%Y-%m-%dT%H:%M:%S'
       )
```
Error 
```
TypeError: __init__() got multiple values for keyword argument 'validators'
```

**Comment by petebrowne** [comment]
If you pass in validators do you think it should:

1. override the default - `[validators.RequiredValidator()]`
2. or append them to the default - `[validators.DateTimeValidator(), validators.RequiredValidator()]`

If we went with option 1, you'd have to explicitly pass the DateTimeValidator. If we went with option 2, which is more convenient, there'd be no way to remove the DateTimeValidator (though I'm not sure what the use case for that would be anyway)

**Comment by joeljames** [comment]
Yes, I agree with you. The DateTimeField should always have the `validators.DateTimeValidator()`. I am leaning towards option2. 
I can submit a quick PR with tests and a patch. 
Let me know if you are fine wth that.

**Comment by petebrowne** [comment]
Yeah go for it, thanks!

**Comment by joeljames** [comment]
@petebrowne 
Below is the PR with fix:
https://github.com/localmed/pyserializer/pull/26

Below is the link to passing build:
https://travis-ci.org/localmed/pyserializer/builds/334220765

Can you please release the version of pyserializer 0.9.1 which contains the fix, if the PR looks good?

**Comment by petebrowne** [comment]
Closed by #26

**Comment by petebrowne** [comment]
Pushed https://pypi.python.org/pypi/pyserializer
